.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish Storybook
+on:
+  push:
+    branches:
+      - develop
+      # TODO: Remove this prior to merging into `develop`
+      - EGRC-275
```

Comment:
The *last* change before merging into develop needs to be to remove this line; I am going to request changes to make sure we keep a little red icon on the PR until this is done. But I also don't want to resolve this too soon because I want to see future changes to this branch still get built/published.